spec/factories.rb

```diff
@@ -11,6 +11,14 @@
     user
     date Time.zone.now
     body 'Entry body'
+
+    trait :with_photo do
+      photo {
+        Rack::Test::UploadedFile.new(
+          Rails.root.join('spec', 'fixtures', 'photo.jpg')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.